runtests.sh

```diff
@@ -77,7 +77,7 @@ run_integration(){
         export SNAPCRAFT=snapcraft-coverage
     fi
 
-    python3 -m unittest discover -b -v -s integration_tests
+    SNAPCRAFT_FROM_SOURCE=1 python3 -m unittest discover -b -v -s integration_tests
```

Comment:
well, not break, but you will have to add the env var to your statement. What we could do is to add an arg to ./runtests integration so you can pass snapcraft.tests.foo. So my idea is that you would use ./runtests if you want to check the source, and adt-run if you want to check the installed.

I didn't want to rely on whether it is installed or not, because I have it installed and sometimes I still want to run it from source, son snapcraft --version or even just "which" are not useful, IMO.

I could have done it the other way, defining SNAPCRAFT_FROM_INSTALLED on debian/tests, but that sounded less intuitive for me.